doc/classes/EditorExportPlatformExtension.xml

```diff
@@ -0,0 +1,283 @@
+				[b]Optional.[/b]
+				Returns number one-click deploy devices (or other one-click option displayed in the menu).
+			</description>
+		</method>
+		<method name="_get_options_tooltip" qualifiers="virtual const">
+			<return type="String" />
+			<description>
+				[b]Optional.[/b]
+				Returns tooltip of the one-click deploy menu button.
+			</description>
+		</method>
+		<method name="_get_os_name" qualifiers="virtual const">
+			<return type="String" />
+			<description>
+				[b]Required.[/b]
+				Returns target OS name.
+			</description>
+		</method>
+		<method name="_get_platform_features" qualifiers="virtual const">
+			<return type="PackedStringArray" />
+			<description>
+				[b]Required.[/b]
+				Returns array of platform specific features.
+			</description>
+		</method>
+		<method name="_get_preset_features" qualifiers="virtual const">
+			<return type="PackedStringArray" />
+			<param index="0" name="preset" type="EditorExportPreset" />
+			<description>
+				[b]Required.[/b]
+				Returns array of platform specific features for the specified [param preset].
+			</description>
+		</method>
+		<method name="_get_run_icon" qualifiers="virtual const">
+			<return type="Texture2D" />
+			<description>
+				[b]Optional.[/b]
+				Returns icon of the one-click deploy menu button, icon should be 16x16 adjusted to the current editor scale, see [method EditorInterface.get_editor_scale].
+			</description>
+		</method>
+		<method name="_has_valid_export_configuration" qualifiers="virtual const">
+			<return type="bool" />
+			<param index="0" name="preset" type="EditorExportPreset" />
+			<param index="1" name="debug" type="bool" />
+			<description>
+				[b]Required.[/b]
+				Returns [code]true[/code] if export configuration is valid.
+			</description>
+		</method>
+		<method name="_has_valid_project_configuration" qualifiers="virtual const">
+			<return type="bool" />
+			<param index="0" name="preset" type="EditorExportPreset" />
+			<description>
+				[b]Required.[/b]
+				Returns [code]true[/code] if project configuration is valid.
+			</description>
+		</method>
+		<method name="_is_executable" qualifiers="virtual const">
+			<return type="bool" />
+			<param index="0" name="path" type="String" />
+			<description>
+				[b]Optional.[/b]
+				Returns [code]true[/code] if specified file is a valid executable (native executable or script) for the target platform.
+			</description>
+		</method>
+		<method name="_poll_export" qualifiers="virtual">
+			<return type="bool" />
+			<description>
+				[b]Optional.[/b]
+				Returns [code]true[/code] if one-click deploy options are changed and editor interface should be updated.
+			</description>
+		</method>
+		<method name="_resolve_platform_feature_priorities" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="preset" type="EditorExportPreset" />
+			<param index="1" name="features" type="PackedStringArray" />
+			<description>
+				[b]Required.[/b]
+				???
```

Comment:
I have no idea what this method supposed to do, or if it should be exposed. It is empty in the all of the existing export platforms.

Reply:
If in doubt, it'd probably be best not to expose it, so that we don't commit ourselves to a broken API. Maintaining compat for virtual methods is harder than normal methods: if you need to change the function signature, you need to make a new method with a new name.

If we don't expose it, and we end up finding a need for it later, we can expose it then (and hopefully at that point, we'll understand it better and be able to give it the right API).